docs/api/zh/textures/Texture.html

```diff
@@ -54,6 +54,14 @@ <h3>[property:Image image]</h3>
 		并在视频播放时不断地更新这个纹理贴图。——[page:VideoTexture VideoTexture] 类会对此自动进行处理。
 		</p>
 
+		<h3>[property:Boolean isTexture]</h3>
+		<p>
+			用于测试这个类或者派生类是否为Texture，默认为*true*。<br /><br />
```

Comment:
Have you already translated this? ;)

Reply:
I don't speak chinese, but I copied it from another documentation page, assuming it is the same everywhere for this property ? Anyway I can remove chinese translations if you wish.

Reply:
I've translated the text out of curiosity with google-translate and it seems to be fine. 

In general, I would avoid doing copy/paste and let the Chinese devs handle the translation.